src/app.js

```diff
@@ -43,9 +44,10 @@ if (LEAD_TIME) {
 }
 
 const metricsOptions = {};
-if (METRICS_PATH) {
-	metricsOptions.metricsPath = METRICS_PATH;
+if (Configuration.has('PROMETHEUS__METRICS_PATH')) {
+	metricsOptions.metricsPath = Configuration.get('PROMETHEUS__METRICS_PATH');
 }
+metricsOptions.durationBuckets = Configuration.data.PROMETHEUS.DURATION_BUCKETS_SECONDS;
```

Comment:
Why not over get() is this notation why documented?

Reply:
As arrays can't be fetched via get currently. Opened as issue ARC-118